syne_tune/tuning_status.py

```diff
@@ -16,11 +16,7 @@
 from collections import defaultdict, OrderedDict
 from typing import List, Dict, Tuple
 
-import pandas as pd
-
 from syne_tune.backend.trial_status import Status, Trial
-import numpy as np
```

Comment:
I am not sure why we would remove this since we assume we have numpy in all our dependencies?